packages/block-editor/src/components/list-view/block-select-button.js

```diff
@@ -7,6 +7,7 @@ import clsx from 'clsx';
  * WordPress dependencies
  */
 import {
+	Badge,
```

Comment:
We currently don't expose `Badge` as a public component. You'll have to use the private API unlock mechanism to use it, see #68062 for a similar example.